.github/workflows/release.yml

```
@@ -0,0 +1,90 @@
+          npx @sentry/cli releases finalize vikings-eventmgmt-mobile@$VERSION
+
+      - name: 🐙 Create GitHub Release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          body: |
+            ## 🔄 Changes
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)